### ISSUE TITLE
Compiling bug

### ISSUE THREAD

**Issue by skalahonza** [JamesRandall/FunctionMonkey#2]
I was follwing steps in this tutorial. I tried the build the app and I ended up with error.

> Severity	Code	Description	Project	File	Line	Suppression State
Error	MSB3073	The command "dotnet "C:\Users\Jan Skála\.nuget\packages\functionmonkey.compiler\0.10.1-beta000\build\netstandard1.0\..\..\tools\netcoreapp2.0/FunctionMonkey.Compiler.dll" C:\Users\Jan Skála\Documents\Visual Studio 2017\Projects\Talxis\Functions\Premiere.Function\bin\Debug\netstandard2.0\bin\Premiere.Function.dll" exited with code -532462766.	Premiere.Function	C:\Users\Jan Skála\.nuget\packages\functionmonkey.compiler\0.10.1-beta000\build\netstandard1.0\FunctionMonkey.Compiler.targets	10	

Output:
> Unhandled Exception: System.FormatException: String was not recognized as a valid Boolean.
1>   at System.Boolean.Parse(String value)
1>   at FunctionMonkey.Compiler.Program.Main(String[] args) in D:\wip\myOpenSource\functionMonkey\Source\FunctionMonkey.Compiler\Program.cs:line 22

## Nugets:
* FunctionMonkey 0.10.0-beta000
* FunctionMonkey.Compiler 0.10.1-beta000

**Comment by JamesRandall** [comment]
Thanks for getting in touch!

Could you share the code for the command you are basing the function on please?

**Comment by skalahonza** [comment]
Sorry, I forgot to add a link for the tutorial. I follwoed this one: https://www.azurefromthetrenches.com/build-elegant-rest-apis-with-azure-functions/

**Comment by JamesRandall** [comment]
There's been a few changes since my original blog post (not sure why you're getting that error though) and its properly documented over here so worth checking this:
http://functionmonkey.azurefromthetrenches.com/guides/gettingStarted.html
I'll update the blog post with a pointer to the new docs.

**Comment by JamesRandall** [comment]
Did the new docs help?

**Comment by skalahonza** [comment]
Unfortuately not 
I followed the instructions and try to compile hello world example.

I got this:
`1>------ Build started: Project: Monkey, Configuration: Debug Any CPU ------
1>Monkey -> C:\Users\Jan Skála\Documents\Visual Studio 2017\Projects\Talxis\Functions\Monkey\bin\Debug\netstandard2.0\bin\Monkey.dll
1>
1>Unhandled Exception: System.FormatException: String was not recognized as a valid Boolean.
1>   at System.Boolean.Parse(String value)
1>   at FunctionMonkey.Compiler.Program.Main(String[] args) in D:\wip\myOpenSource\functionMonkey\Source\FunctionMonkey.Compiler\Program.cs:line 22
1>C:\Users\Jan Skála\.nuget\packages\functionmonkey.compiler\0.10.1-beta000\build\netstandard1.0\FunctionMonkey.Compiler.targets(10,5): error MSB3073: The command "dotnet "C:\Users\Jan Skála\.nuget\packages\functionmonkey.compiler\0.10.1-beta000\build\netstandard1.0\..\..\tools\netcoreapp2.0/FunctionMonkey.Compiler.dll" C:\Users\Jan Skála\Documents\Visual Studio 2017\Projects\Talxis\Functions\Monkey\bin\Debug\netstandard2.0\bin\Monkey.dll" exited with code -532462766.
1>Done building project "Monkey.csproj" -- FAILED.`
If you want, I can zip you the project.

**Comment by JamesRandall** [comment]
If you could zip the project for me that would be great! Thank you!

**Comment by skalahonza** [comment]
[Monkey.zip](https://github.com/JamesRandall/FunctionMonkey/files/2213907/Monkey.zip)
Here it is. Maybe I totally messed something up, but I tired it many times.

**Comment by JamesRandall** [comment]
Thank you for that - I've found the issue! Your project worked for me and so I compared my command line with yours and realised that you are using the Visual Studio 2017 folder as part of your path. I never use that and always use something like d:\wip. The space in the path caused the command line arguments to get thrown off.

If you update the FunctionMonkey.Compiler package it should now work for you.

Thanks for your help - because of how I set up my environments it would have been super hard for me to find this on my own.
